sunkit_image/granule.py

```diff
@@ -0,0 +1,298 @@
+"""
+This module contains functions that will segment images for granule detection.
+"""
+import logging
+
+import numpy as np
+import scipy.ndimage as sndi

```

Comment:
For the sake of readability, I'd avoid this alias.

Reply:
I assume you are refering to the `sndi` alias? I will remove that.

Reply:
Yep, I was referring to `sndi`. Sorry for the ambiguity there.